README.md

```diff
@@ -43,11 +43,11 @@ All URLs provided must be relative, apart from locally, where localhost is allow
 {
     "continueUrl" : "/test",
     "businessVerificationCheck" : true ,
-    "optServiceName" : "Service Name",
+    "optServiceName" : "Service Name", // deprecated, use labels.en.optServiceName
     "deskProServiceId" : "abc",
     "signOutUrl" : "/sign-out",
     "accessibilityUrl" : "/accessibility-statement/my-service",
-    "optFullNamePageLabel" : "What is the name of the nominated partner?",
+    "optFullNamePageLabel" : "What is the name of the nominated partner?", // deprecated, do not use 
```

Comment:
Could this be "deprecated, use labels.en.optFullNamePageLabel" to match the first comment. But happy to approve either way